topdown/eval.go

```diff
@@ -2791,6 +2795,10 @@ func (e evalVirtualComplete) evalValue(iter unifyIterator, findOne bool) error {
 		return err
```

Comment:
☝️ Interesting. Doesn't this mean that we're _not_ caching the rule result when it's been the `default` rule that hit? Feels like a thing we could do, though. (Not for this PR, obviously.)